components/Starknet/modules/tools/pages/starkgate-cancelling a deposit.adoc

```diff
@@ -16,7 +16,7 @@ In order to guard against an attack, it takes approximately five days to cancel
 
 . To cancel a deposit, call the `depositCancelRequest` request function.
 +
-When StarkGate receives the cancellation request, a counter begins to count five days.
+* When StarkGate receives the cancellation request, a counter begins to count five days.
```

Comment:
General rule: Never have a single list item. If you have one bullet, you need at least one more. If you only have on item, don't use a bullet.

Treat bullets and numbers as structural, not as formatting.